sponsoring/index.md

```diff
@@ -24,8 +24,15 @@ Or with stuff like:
 
 If you're interested in these options, please get in touch and email the Project Management Committee at pmc@voxpupuli.org.
 
-At the moment we've multiple sponsors via GitHub. In addition to that,
-[Hetzner Cloud GmbH](https://www.hetzner-cloud.de/en) sponsors us cloud resources.
+At the moment we've multiple sponsors via GitHub. In addition to that, multiple
+companies sponsor our cloud/CI resources.
+
+Thanks GitHub for sponsoring us 60 concurrent CI runners and the
```

Comment:
Aren't the 60 concurrent CI runners part of the team plan? I was expecting it to be listed on https://github.com/pricing but I can't find it.

Reply:
my understanding is that it's not part of it, but their pricing is complicated.